Set an empty array when web property has no profile

5t111111 opened the pull request:
I've add a line to simply set an empty array for avoiding an exception by invoking methods on `nil` .

Since you can remove all profiles from a web property in Google Analytics console, an exception [ `NoMethodError (undefined method 'map' for nil:NilClass` ] might occur on fetching account summary.

```ruby
        WebProperty.new(attributes, user).tap { |web_property|
          web_property.account = account
          web_property.profiles = profiles.map { |profile| # <= NoMethodError on this line
            profile['accountId'] = account.id
            profile['webPropertyId'] = web_property.id
            Profile.build_from_summary(profile, user, account, web_property)
          }
        }
```

Sija commented on lib/legato/management/web_property.rb:
```diff
@@ -42,6 +42,7 @@ def self.from_child(child)
 
       def self.build_from_summary(attributes, user, account)
         profiles = attributes.delete('profiles') || attributes.delete(:profiles)
+        profiles = [] unless profiles
```
`profiles ||= []`

tpitale commented:
The first assignment of profiles could just add `|| []`

Also, can you check the failing specs?

5t111111 commented:
@tpitale I've changed to just add `|| []` to the first assignment of profiles, and also added some specs to check if it works when there a web property without any profile exists.

The cause of failing specs is an installing problem of Rake which requires Ruby 1.9.3 or later. I don't know if fixing Rake version to more earlier version could be the solution, but as Rake is only development dependency, I think it's ok to just remove Ruby 1.9.2 from `.travis.yml`

tpitale commented:
Great, I removed 1.9.2 and added 2.2.2 to travis config. May add 2.3 shortly.
